### PR TITLE
Made release note of how to use LED_ONBOARD constant.

### PR DESCRIPTION
**Pull request by zfields**
Changed OnBoardLED sample to reflect proper usage of LED_ONBOARD.

### CODE REVIEW

File: OnBoardLED.md [hunk 1]
```diff
@@ -5,46 +5,40 @@ permalink: /OnBoardLED.htm
 ---
 
 # On Board LED
-Learn how to use the on board thermal sensor.
+Learn how to toggle the GP Led on the Galileo on and off.
```
Comment (JustAddWires) [inline OnBoardLED.md]: I think this should be "on-board", GP may not immediately communicate to people.

Comment (anthony-ngu) [inline OnBoardLED.md]: we may have to point out which one is considered the on-board LED if we do that. (GP is labeled on the board)

Comment (JustAddWires) [inline OnBoardLED.md]: Ok, how about referring to it as the on-board LED in the item that is selected to get you to the sample, and then: “on-board LED (labeled “GP” on the board)” in the sample itself?

**Comment by zfields** [comment]
All instances of LED_BUILTIN have been removed from check-in favor of LED_BUILTIN, only persists in commit descriptions

**Comment by anthony-ngu** [comment]
Since both Bert and I think this looks good, I will merge it!

**Pull request merged**